src/data.js

```diff
@@ -31,6 +40,21 @@ genotet.data.serverURL;
 genotet.data.uploadURL;
 
 /**
+ * User information are saved to this URL via http and received via jsonp.
```

Comment:
Umm I didn't notice this earlier. But technically user information is not part of "data". We can create a new namespace called genotet.user for all user related information and login/logout/authentication functions.